packages/cli/src/wait-tracker.ts

```diff
@@ -141,7 +143,7 @@ export class WaitTracker {
 	}
 
 	stopTracking() {
-		this.logger.debug('Wait tracker shutting down');
+		this.logger.debug('Shutting down');
```

Comment:
This now logs `executions Shutting down`. I think something like this might make more sense:
```suggestion
		this.logger.debug('Shutting down wait tracking');
```